source/content/guides/drupal/drupal-hosted-createempty-md/05-composer-upstream.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Migrate a Site That Was Created With an Empty Upstream to Drupal 9
+title: Migrate a Site That Was Created With an Empty Upstream to the Latest Version of Drupal
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createempty-md/composer-upstream